Gems/AudioSystem/Code/Source/Engine/SoundCVars.cpp

```diff
@@ -131,10 +131,17 @@ namespace Audio::CVars
         "The number of audio objects to preallocate in a pool.\n"
         "Usage: s_AudioObjectPoolSize=" AZ_TRAIT_AUDIOSYSTEM_AUDIO_OBJECT_POOL_SIZE_DEFAULT_TEXT "\n");
 
+#if defined (CARBONATED)
+    AZ_CVAR(float, s_PositionUpdateThreshold, 1.0f,
+        nullptr, AZ::ConsoleFunctorFlags::Null,
+        "An audio object needs to move by this distance in order to issue a position update to the audio system.\n"
+        "Usage: s_PositionUpdateThreshold=5.0\n");
+#else
     AZ_CVAR(float, s_PositionUpdateThreshold, 0.1f,
         nullptr, AZ::ConsoleFunctorFlags::Null,
         "An audio object needs to move by this distance in order to issue a position update to the audio system.\n"
         "Usage: s_PositionUpdateThreshold=5.0\n");
+#endif
```

Comment:
This seems like a really large change (1 meter) - any reason not to keep 0.1 and just fix the gating above?

Reply:
Tried different values for s_PositionUpdateThreshold using the Remote console. 
Value 0.6 is good for me on iPhone14. Value 0.5 is good too for me but slightly less.
I think we could ask QA to find a mostly suitable value for s_PositionUpdateThreshold using the Remote console, maybe 0.4 or 0.3 is enough for QA. 
@Twolewis ^^^

Reply:
Ok, interesting.  Sure, let's leave at that and see if anyone complains.  0.6 is still _a lot_ but alright

Reply:
I have checked.
 If I set s_PositionUpdateThreshold to 0.1 via Remote console, then issue MAD-15711 returns back. 
 The fix is to increase s_PositionUpdateThreshold value. 
 Issue starts to disappear after s_PositionUpdateThreshold is 0.3 or greater but I don't know, which s_PositionUpdateThreshold is good for QA.